src/test/java/io/opentelemetry/sdk/testing/exporter/InMemorySpanExporter.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Use InMemorySpanExporter for testing.

This code is copied from
https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/testing/src/main/java/io/opentelemetry/sdk/testing/exporter/InMemorySpanExporter.java